client/rpc/src/lib.rs

```diff
@@ -168,7 +168,7 @@ pub fn error_on_execution_failure(reason: &ExitReason, data: &[u8]) -> Result<()
 	match reason {
 		ExitReason::Succeed(_) => Ok(()),
 		ExitReason::Error(e) => {
-			if *e == ExitError::OutOfGas || *e == ExitError::OutOfFund {
+			if *e == ExitError::OutOfGas {
 				// `ServerError(0)` will be useful in estimate gas
 				return Err(Error {
 					code: ErrorCode::ServerError(0),
```

Comment:
Also L175 -- should be just `format!("out of gas")`

Reply:
Done